nixos/modules/services/system/dbus.nix

```diff
@@ -184,6 +184,11 @@ in
         aliases = [
           "dbus.service"
         ];
+        unitConfig = {
+          # We get errors when reloading the dbus-broker service
+          # if /tmp got remounted after this service started
+          RequiresMountsFor = [ "/tmp" ];
```

Comment:
Could you give an example of those errors in this PR? Is this the total set of mounts required?

Reply:
Sure, I added some more info in the description above.

Reply:
That's enough for me to say yes.